src/pages/settings/Subscription/SubscriptionSettingsPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import Navigation from '@libs/Navigation/Navigation';
+
+function SubscriptionSettingsPage() {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const {translate} = useLocalize();
+
+    return (
+        <ScreenWrapper testID={SubscriptionSettingsPage.displayName}>
+            <HeaderWithBackButton
+                title={translate('workspace.common.subscription')}
+                onBackButtonPress={() => Navigation.goBack()}
```

Comment:
should this specifically go to `ROUTES.SETTINGS`?

Reply:
Looking at some other setting screens, it's `Navigation.goBack(navigateBackTo)` for **ProfilePage.tsx** (`navigateBackTo` is an optional route param) and `Navigation.goBack()` for **PreferencesPage.tsx**, that's what I based my solution upon. Do you think we should specify which screen it should go back to?

Reply:
I don't feel super strongly, let's merge as-is and we can adjust later if we want